mono/metadata/loader.c

```diff
@@ -2620,16 +2620,17 @@ mono_method_get_token (MonoMethod *method)
 	return method->token;
 }
 
-MonoMethodHeader*
+MONO_API MonoMethodHeader*
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.

Reply:
Again this was very deliberate, but yes maybe redundant.